src/components/protected-areas-layers/protected-areas-layers-component.jsx

```diff
@@ -25,12 +25,7 @@ const ProtectedAreasLayers = ({ handleGlobeUpdating, handleLayerToggle, activeLa
   const toggleLayer = (layersPassed, option) => {
```

Comment:
I was talking about this outer `toggleLayer` function, not the inner `layerToggled`. As you can see there is more shared logic inside of it (updating the flag for displaying the loader, analytic events... )

Reply:
right 🤦‍♀ on it